### PR TITLE
Rename and reorder table columns

### PR DESCRIPTION
**Pull request by logstar**
Renamed and reordered table columns, according to suggestions by @taylordm @chinwallaa @afarrel @runjin326 .

### CODE REVIEW

**Review by runjin326 — APPROVED** [review]
@logstar, @chinwallaa , thanks for preparing this! I reviewed the changes - they are what we've discussed and make sense to me. No typo found. LGTM!

File: front-end/page_target/SnvByVariant_Config.json [hunk 1]
```diff
@@ -159,8 +159,8 @@
   },
   {
     "id": "OncoKBOncogeneTSG",
-    "label": "OncoKB oncogene TSG",
+    "label": "OncoKB Oncogene/TSG",
```
Comment (jonkiky) [inline front-end/page_target/SnvByVariant_Config.json:162]: For the "/" may can cause the confusion  as "/" present the mathematical meaning throughout the configuration file. 
How about to change it to " OncoKB Oncogene | TSG " or  "OncoKB Oncogene Or TSG" ?

**Comment by Nahomtes** [comment]
This PR LGTM!

**Review by jonkiky — APPROVED** [review]
LGTM!

**Pull request merged**